### PR TITLE
NodeGraph: Display node graph collapsed by default with trace view

### PR DESCRIPTION
**Pull request by aocenas**
When showing node graph together with trace view for Tempo, Jaeger and Zipkin it may create perf issues with lots of spans and also may not be the main thing user wants to see so this default to node graph being collapsed and user can expand if needed.

Also adds a message in case there is more than 1k nodes:
Screenshot from 2021-05-25 15-24-29

### CODE REVIEW

**Review by torkelo — APPROVED** [review]
Nitpick: 
The colloapse angle-down icon looks a bit wrong padding wise (too far to the right) so the two box headings do not align

**Comment by aocenas** [comment]
@torkelo 
> Nitpick:
> The colloapse angle-down icon looks a bit wrong padding wise (too far to the right) so the two box headings do not align

I did align it but with some negative margins :man_shrugging:. Seems like icons are not always alligned to the actual svg, ie they do not start and end at the edge of the svg so not sure how else to align these:
Screenshot from 2021-05-25 18-15-56
Screenshot from 2021-05-25 18-13-49

File: public/app/features/explore/NodeGraphContainer.test.tsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        splitOpen={(() => {}) as any}
+      />
+    );
+
+    expect(container.firstChild?.childNodes.length).toBe(2);
+    expect(container.querySelector('svg')).toBeInTheDocument();
+  });
+});
+
+const range = {
```
Comment (Elfo404) [inline public/app/features/explore/NodeGraphContainer.test.tsx:38]: nit: you can also use `getDefaultTimeRange` from `@grafana/data`

File: public/app/features/explore/NodeGraphContainer.tsx [hunk 2]
```diff
@@ -1,38 +1,59 @@
...
 }
 export function UnconnectedNodeGraphContainer(props: Props & ConnectedProps<typeof connector>) {
-  const { dataFrames, range, splitOpen, short } = props;
+  const { dataFrames, range, splitOpen, withTraceView } = props;
   const getLinks = useLinks(range, splitOpen);
+  const styles = useStyles2(getStyles);
 
-  const [open, setOpen] = useState(true);
+  const { nodes } = useCategorizeFrames(dataFrames);
+  const [open, setOpen] = useState(false);
```
Comment (Elfo404) [inline public/app/features/explore/NodeGraphContainer.tsx:35]: SUPERNIT: we also have `useToggle` from `react-use`

File: public/app/features/explore/NodeGraphContainer.tsx [hunk 3]
```diff
@@ -1,38 +1,59 @@
...
+  const countWarning =
+    withTraceView && nodes[0]?.length > 1000 ? (
+      <span className={styles.warningText}> ({nodes[0].length} nodes, can be slow to load)</span>
+    ) : (
+      ''
```
Comment (Elfo404) [inline public/app/features/explore/NodeGraphContainer.tsx:40]: This will create a text node, did you consider using `null` instead?

**Review by Elfo404 — APPROVED** [review]
Code looks awesome! 🚀 

The only "issue" is with `countWarning` returning `''` instead of `null`, but shouldn't be a major issue

**Pull request merged**